newdle_frontend.js

```diff
@@ -3,17 +3,25 @@ const tick_string = "tick for now"
 
 const cross_string = "cross for now"
 
+const API_info = {headline: "Boris Johnson says tens of thousands could be sent to Rwanda under relocation plan for asylum seekers - UK politics live - The Guardian", scrambled_headline: "Bsrio onnJsoh asys snte fo onduathss uolcd be nset ot awnaRd rneud erolniocta pnla rof mluyas sesekre - UK iosplcti ievl - eTh andiuarG"}
```

Comment:
Constants should be written in uppercase: https://www.w3schools.com/js/js_conventions.asp